test/ensureArray.test.ts

```diff
@@ -0,0 +1,9 @@
+import { ensureArray } from '../src/ensureArray';
+
+it('any value passed should return array', () => {
+  const arrayWithTestSting = ensureArray(['test']);
+  const testString = ensureArray('test');
+
+  expect(arrayWithTestSting).toEqual(['test']);
+  expect(testString).toEqual(['test']);
+});
```

Comment:
You are testing two different cases here:
1. It should return an array with the given value
2. It return the given value if it is an array

I would advise separating these into two different tests - that way the intent is much clearer on what is happening :)

Reply:
So it should the two `it`, right? also in the second one where i test for `It should return an array with the given value` should i add more primitive types to test or only one is enough? 
like: 
```js
it('should return an array with the given value', () => {
  const testString = ensureArray('test');
  const testNumber = ensureArray(0);
  const testBoolean = ensureArray(true);

  expect(testString).toEqual(['test']);
  expect(testNumber).toEqual([0]);
  expect(testBoolean).toEqual([true]);
});
```
or only:
like: 
```js
it('should return an array with the given value', () => {
  const testString = ensureArray('test');

  expect(testString).toEqual(['test']);
});
```

Reply:
Hmm... Good question! Based on the fact that the `ensureArray` method is based on `any` type... I would check:
* A primitive
* A function
* An object

Between those three, I'd say you'd have it covered

Reply:
@nobrayner Yeah, makes sense. Updated the test add the assertions for those 3 different value types. let me know if need to change something else 👍